docs/02-StewardForValidators.md

```diff
@@ -0,0 +1,75 @@
+# Validators Instructions for Setting Up Steward
+
+Steward is an application intended for developers and validators on the Sommelier network.
+
+It can run as a server for forearding cellar contract calls from Strategy Providers through the Sommelier chain, or it can run in test mode to directly interact with Ethereum contracts as a single signer.
+
+It integrates the full functionality of gorc for operating as an orchestrator and relayer of [Gravity bridge](https://github.com/PeggyJV/gravity-bridge/) messages between the Ethereum and Cosmos chains.
+
+Stewards works in conjunction with the Orchestrator, so both processes must be running to fully participate in Cellar management.
+
+## Quickstart
+
+To start Steward, simply run
+
+```bash
+steward -c [path to your config toml] start
+```
+
+## Setting Up Steward
+
+In this section, let’s explore setting up steward for validators. There are two ways that validators will use Steward:
+
+1. Running Steward as a gRPC server for relaying SP contract calls through the chain. If a validator is not running this server, it cannot participate in Cellar management.
+2. Manually scheduling contract calls when the validator set needs to coordinate a function  call on a cellar. An example of this would be transferring a Cellar’s accrued fees to a module account on chain.
```

Comment:
I believe SPs will also be doing this (they don't need on-chain scheduling because they have an external clock they can follow). Scheduled calls are for when multiple parties need to coordinate for valset consensus, like shutting down a cellar.

Reply:
It shouldn't require a scheduled cork to transfer accrued fees.

Reply:
Oh I get what you mean now. K